README.md

```diff
@@ -1,38 +1,53 @@
-What is Notepad++ ?
-===================
+# What is Notepad++ ?
 
 [![Join the chat at https://gitter.im/notepad-plus-plus/notepad-plus-plus](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/notepad-plus-plus/notepad-plus-plus?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 Notepad++ is a free (free as in both "free speech" and "free beer") source code editor and Notepad replacement that supports several programming languages and natural languages. Running in the MS Windows environment, its use is governed by GPL License.
 
 
-To build Notepad++ package from source code:
-============================================
+## To build Notepad++ from source:

```

Comment:
Do you also want the old style for "To build `notepad++.exe`:", "To build `SciLexer.dll` with boost:", and "To build `SciLexer.dll` *without* boost:"?